.gitignore

```diff
@@ -16,4 +16,5 @@
 # Fonto's proprietary files
 # It is recommended that you commit these files to your version control. To this end, please remove the following line
 # from this file:
-/platform/
\ No newline at end of file
+/platform/
+/platform-linked/
```

Comment:
`platform-linked` is not a term familiar to partners, I'd rather not change this file in this PR